.taskcluster.yml

```diff
@@ -41,7 +41,6 @@ tasks:
             - "-cx"
             - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py &&
                rustup toolchain install stable &&
-               rustup toolchain install nightly &&
                rustup default stable &&
```

Comment:
I wonder if this line is still needed.

Reply:
Since tests are passing it's not needed anymore :)

Reply:
Well, the line is still there :P

Reply:
Ouch, you're right!